rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -1355,7 +1356,9 @@ def get_module_answers(ro_dao, module, p_id, qr_id=None, return_responses=False)
                                 ON qra.question_id = qq.questionnaire_question_id
                      INNER JOIN questionnaire q
                                 ON qq.questionnaire_id = q.questionnaire_id
-            WHERE qr.questionnaire_response_id = :qr_id;
+            WHERE qr.questionnaire_response_id = :qr_id
+            -- Order by question and calculated answer so any duplicates from the same response are adjacant
+            ORDER BY qra.question_id, 4
```

Comment:
Would it be better to use the column name rather than the number in the `order by` to make it clearer what is getting ordered?

Reply:
Yes, I should've caught that I could use that alias/calculated column name from the inner select.  Thanks!